jdl/converters/jdl-to-json/jdl-with-applications-to-json-converter.spec.ts

```diff
@@ -1110,6 +1110,8 @@ JSONEntity {
                   global: {
                     custom: 42,
                   },
+                  source: {},
```

Comment:
The use of a record in the convert method forces to have the 3 defined. wouldn't it be better to be able to specify a partial here?